third-party/realdds/include/realdds/topics/notification/notification.idl

```diff
@@ -7,14 +7,17 @@ module realdds
     {
         module raw
         {
-            module device
```

Comment:
Notification topic name does not include the device name? e.g. /realsense/D455/notification?

Reply:
Topic name? Not sure what you mean. This is a general-purpose notification topic. What does it have to do with a device?